test/GammaPoolERC721.behavior.ts

```diff
@@ -480,7 +480,7 @@ export function shouldBehaveLikeERC721(errorPrefix: any) {
 
       context('when the operator is the owner', function () {
         it('reverts', async function () {
-          await expect(this.token.setApprovalForAll(this.owner.address, true, { from: this.owner.address })).to.be.revertedWith('ERC721: approve to caller');
+          await expect(this.token.setApprovalForAll(this.owner.address, true, { from: this.owner.address })).to.be.revertedWith('ERC721: FORBIDDEN');
         });
```

Comment:
what is the point of using `context` instead of `describe`? Let's stick to just `describe` as it brings a lot of confusion and inconsistency